src/frontend/src/binder/expr/function.rs

```diff
@@ -839,6 +839,10 @@ impl Binder {
                 ("array_sum", raw_call(ExprType::ArraySum)),
                 ("array_position", raw_call(ExprType::ArrayPosition)),
                 ("array_positions", raw_call(ExprType::ArrayPositions)),
+                ("array_contains", raw_call(ExprType::ArrayContains)),
+                ("arraycontains", raw_call(ExprType::ArrayContains)),
+                ("array_contained", raw_call(ExprType::ArrayContained)),
+                ("arraycontained", raw_call(ExprType::ArrayContained)),
```

Comment:
Seems that these functions are not provided by PG, do we really need to expose these functions to our users? BTW, even if we decide to expose them, `arraycontains` and `arraycontained` seem unnecessary and we need to think about potential inalignment with PG if PG adds these functions with different signatures in the future.